FWCore/PluginManager/src/pauseMaxMemoryPreload.h

```diff
@@ -0,0 +1,9 @@
+#ifndef FWCore_PluginManager_src_pauseMaxMemoryPreload_h
+#define FWCore_PluginManager_src_pauseMaxMemoryPreload_h
+
+// By default do nothing, but add "hooks" that MaxMemoryPreload can
+// override with LD_PRELOAD
+void pauseMaxMemoryPreload();
+void unpauseMaxMemoryPreload();
```

Comment:
How about designing a sentinel class here to auto unpause?

Reply:
Done